### ISSUE TITLE
Add/edit item inventory

### ISSUE THREAD

**Issue by Melissa1008** [andrewimpellitteri/awning_wo#50]
Order / tab order should be:

Quantity
Description
Material
Condition
Color
Size/Weight
Price
